modules/workflows/OrcaCopyToGlacierWorkflow/orca_copy_to_glacier_workflow.asl.json

```diff
@@ -7,9 +7,10 @@
         "cma": {
           "event.$": "$",
           "task_config": {
-            "multipart_chunksize_mb": "{$.meta.collection.meta.multipart_chunksize_mb}",
             "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
+            "multipart_chunksize_mb": "{$.meta.collection.meta.multipart_chunksize_mb}",

```

Comment:
I am wondering if we shouldn't change the setting here from `"{$.meta.collection.meta.multipart_chunksize_mb}"` to `"{$.meta.collection.meta.s3MultipartChunksizeMb}"` so users only have to set this value in one place instead of two. Ideally we would also change from `multipart_chunksize_mb` to `s3MultipartChunksizeMb` as well.

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-338